app/src/main/res/layout/browser_overlay.xml

```diff
@@ -8,13 +8,19 @@
     android:background="@color/browser_overlay_background"
     android:orientation="vertical"
     android:descendantFocusability="afterDescendants"
-    android:padding="20dp">
+    android:paddingTop="27dp"
+    android:paddingBottom="27dp"
+    >
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:layout_weight="1"
-        android:orientation="horizontal">
+        android:layout_height="wrap_content"
+        android:layout_marginStart="57dp"
+        android:layout_marginEnd="57dp"
```

Comment:
do we care about putting these into constants?

Reply:
No – jumping between multiple files to read constant values is a huge pain (and it pollutes the xml files they're defined in). It's useful to extract values when:
- They're used many, many times
- They change across device configurations
- They're needed in code